src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Unit_goal.json

```diff
@@ -0,0 +1,14431 @@
+      "name": "Unit",
+      "isArray": false
+    },
+    "targetClass": "net.sf.freecol.common.model.Unit",
+    "isVarArgs": false,
+    "returnType": {
+      "qualifiedName": "boolean",
+      "name": "boolean",
+      "isArray": false
+    },
+    "parameters": [],
+    "paramTags": [],
+    "returnTag": {
+      "comment": "True if this Unit can carry treasure.",
+      "kind": "RETURN",
+      "condition": ""
+    },
+    "throwsTags": []
+  },
+  {
+    "signature": "canCaptureGoods()",
+    "name": "canCaptureGoods",
+    "containingClass": {
+      "qualifiedName": "net.sf.freecol.common.model.Unit",
+      "name": "Unit",
+      "isArray": false
+    },
+    "targetClass": "net.sf.freecol.common.model.Unit",
+    "isVarArgs": false,
+    "returnType": {
+      "qualifiedName": "boolean",
+      "name": "boolean",
+      "isArray": false
+    },
+    "parameters": [],
+    "paramTags": [],
+    "returnTag": {
+      "comment": "True if this Unit is capable of capturing goods.",
+      "kind": "RETURN",
+      "condition": ""
+    },
+    "throwsTags": []
+  },
+  {
+    "signature": "isTradingUnit()",
+    "name": "isTradingUnit",
+    "containingClass": {
+      "qualifiedName": "net.sf.freecol.common.model.Unit",
+      "name": "Unit",
+      "isArray": false
+    },
+    "targetClass": "net.sf.freecol.common.model.Unit",
+    "isVarArgs": false,
+    "returnType": {
+      "qualifiedName": "boolean",
+      "name": "boolean",
+      "isArray": false
+    },
+    "parameters": [],
+    "paramTags": [],
+    "returnTag": {
+      "comment": "True if this is a trading unit.",
+      "kind": "RETURN",
+      "condition": ""
+    },
+    "throwsTags": []
+  },
+  {
+    "signature": "isColonist()",
+    "name": "isColonist",
+    "containingClass": {
+      "qualifiedName": "net.sf.freecol.common.model.Unit",
+      "name": "Unit",
+      "isArray": false
+    },
+    "targetClass": "net.sf.freecol.common.model.Unit",
+    "isVarArgs": false,
+    "returnType": {
+      "qualifiedName": "boolean",
+      "name": "boolean",
+      "isArray": false
+    },
+    "parameters": [],
+    "paramTags": [],
+    "returnTag": {
+      "comment": "True if this unit is a colonist.",
+      "kind": "RETURN",
+      "condition": ""
+    },
+    "throwsTags": []
+  },
+  {
+    "signature": "isCarrier()",
+    "name": "isCarrier",
+    "containingClass": {
+      "qualifiedName": "net.sf.freecol.common.model.Unit",
+      "name": "Unit",
+      "isArray": false
+    },
+    "targetClass": "net.sf.freecol.common.model.Unit",
+    "isVarArgs": false,
+    "returnType": {
+      "qualifiedName": "boolean",
+      "name": "boolean",
+      "isArray": false
+    },
+    "parameters": [],
+    "paramTags": [],
+    "returnTag": {
+      "comment": "True if this unit can carry goods or other units.",
+      "kind": "RETURN",
+      "condition": "target.unitType.canCarryGoods() || target.unitType.canCarryUnits()"
```

Comment:
I think the comment should be translated as
`target.unitType.canCarryGoods() || target.unitType.canCarryUnits() ? result==true`